test/functional/bundleadd/add-include.bats

```diff
@@ -43,15 +43,18 @@ test_setup() {
 
 @test "ADD048: Adding a bundle that includes another bundle (no tracking state)" {
 
+	# simulate as if test-bundle3 is already installed
+	sudo touch "$TARGETDIR"/usr/share/clear/bundles/test-bundle3
```

Comment:
maybe testlib func: create_dir

Reply:
in this case I'm creating a file, not a directory, and even if it was a directory the `create_dir` function in testlib does something more complicated than just creating a directory, it creates a directory, gets its hash and creates a tar for it.

Reply:
Okay.